woocommerce/woocommerce-gutenberg-products-block
# Store Customization > Replace patterns based on the Products (Beta) block with an equivalent based on the Product Collection block

## What

On this PR, four patterns that originally relied on the now-retired **Products (beta)** block were replaced with an equivalent based on the new **Product Collection** block.

In addition to the block replacement, the width for all patterns was changed from **none** to **wide** so it complies with the expected for the templates within the pattern assembler.

| Old Pattern | Replacement |
| ------ | ----- |
|  3-Column Product Row <img width="686" alt="Screenshot 2023-10-04 at 17 12 47" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/2c18c4ba-d75a-43b2-aa8d-7e7734363655"> |  Product Collection 3 Columns  <img width="1227" alt="Screenshot 2023-10-04 at 17 13 37" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/6e842ad8-b163-4c02-9900-c6400a039960"> |
| 1:1 Image 4-Column Product Row <img width="685" alt="Screenshot 2023-10-04 at 17 14 25" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/120b128d-b31d-42df-a7de-db128840a9ae"> |  Product Collection 4 Columns <img width="1233" alt="Screenshot 2023-10-04 at 17 14 59" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/5c95339e-0acb-451d-95f5-966b0c22506a"> |
| Minimal 5-Column Product Row <img width="688" alt="Screenshot 2023-10-04 at 17 15 54" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/4b405ace-ef43-4c70-aa0e-0ce9ed268e4e"> |  Product Collection 5 Columns <img width="1229" alt="Screenshot 2023-10-04 at 17 16 32" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/cbfce4ca-cdf6-440d-86e2-f3f44a18e373">  |
|  Featured Products 5-Item Grid  <img width="1215" alt="Screenshot 2023-10-04 at 17 17 40" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/127ccbf4-bbf8-4939-96d4-a97dc9125d26"> |  Product Collection: Featured Products 5 Columns  <img width="1220" alt="Screenshot 2023-10-04 at 17 18 14" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/4703a9df-6ab6-4970-9dbb-f2f35414499f"> |

Fixes #11132

## Why



We need to ensure the new Product Collection block is used to compose the patterns within the Assembler.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new post
2. Insert the following patterns: Product Collection 3 Columns, Product Collection 4 Columns, Product Collection 5 Columns, Product Collection: Featured Products 5 Columns
3. Make sure all of them are using the Product Collection block for rendering in the editor:

<img width="1995" alt="Screenshot 2023-10-04 at 17 23 05" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/9d63321a-fad2-41de-835b-2490ce7f9d27">

4. Save and head over to the front end: make sure things are working as expected and the designs match the ones displayed on the screenshots in this PR.



## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> New Product Collection Patterns: add the new Product Collection 3 Columns, Product Collection 4 Columns, Product Collection 5 Columns and Product Collection: Featured Products 5 Columns patterns
